witness/golang/omniwitness/internal/omniwitness/omniwitness.go

```diff
@@ -67,11 +72,23 @@ type multiLogFeederConfig struct {
 	Logs []config.Log `yaml:"Logs"`
 }
 
+// OperatorConfig allows the bare minimum operator-specific configuration.
+// This should only contain configuration details that are custom per-operator.
+type OperatorConfig struct {
+	WitnessSigner   note.Signer
+	WitnessVerifier note.Verifier
+
+	GithubUser  string
```

Comment:
nit: strictly, I think it should be GitHub :)

Reply:
Yeah, but this is way nicer to read, and easier to identify what the interesting part is than the strictly correct version. Honestly, I find `GitHubUser` a little distressing on its own, and when followed by 2 other similarly named fields it's just too much.

Reply:
Discussed offline. Resolving this.